nautobot/extras/models/jobs.py

```diff
@@ -762,6 +762,13 @@ class ScheduledJob(BaseModel):
         verbose_name="Approval date/time",
         help_text="Datetime that the schedule was approved",
     )
+    custom = models.CharField(
```

Comment:
This name needs to be more descriptive. I would prefer that it be called `custom_schedule` or even just `crontab` so that it's clear that it if it is set, it will be used in that way. On its own `custom` is not quite clear I don't think.